nixos/modules/services/web-apps/siyuan.nix

```
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.siyuan;
+  username = "siyuan";
+  workingDirectory = "/var/lib/siyuan";
+in
+
+{
+  options.services.siyuan = {
+    enable = mkEnableOption "the Siyuan service";
```

```suggestion
    enable = mkEnableOption "the SiYuan service";
```

I think the official name of the package has an upper case `Y`.
This is not the only place where it should be replaced, there are more below this.